classes/Zipcode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Value type wrapping validation logic for zipcode.
+ */
+class Zipcode {
```

Comment:
We should start calling this `postalCode` as this is a more generic and non-specific name for it. "Zip codes" are the US equivalent.

Reply:
Why change it? If we all agree to use this already ubiquitous language. Besides, db fields are already named like this. Renaming is merely a burden without any noticable benefit.

Apart from that, if any further distinguishing is required based on countries it can be reflected in names of validation rules.

Reply:
While things are just small it's easy to make them a bit more generic, but renaming those is also super easy so lets keep it for now